index.html

```diff
@@ -110,6 +110,197 @@ <h1 class="main__title">
         </div>
       </div>
     </section>
+
+
+
+<!-- Block about -->
+    <section class="about">
+      <div class="container">
+        <div class="row">
+
+          <div class="column-lg-7">
+            <div class="about__title">О курсе</div>
+
+            <div class="about__text">
+              На курсе Fullstack-разработки на JavaScript изучаем профессиональные инструменты
+              front-end и back-end разработки React и Node.js, СУБД MySQL и MongoDB. Требования к
+              слушателям на входе: знания html, css, основ программирования на javascript.
+            </div>
+
+            <div class="about__text">
+              Накурсе преподают эксперты из IT-компаний Lad. В конце обучения слушатели курса
+              защищают итоговые проекты. Лучшие получают приглашение поступить на
+              стажировку, в интернатуру или в штат IT-компании.
+            </div>
+
+            <div class="about__text">Мы изучаем:</div>
+
+            <div class="about__box">
+
+              <div class="about__block">
+                <img class="about__image-React" src="img/React.svg" alt="">
+                <p class="about__name">React</p>
+              </div>
+
+              <div class="about__block">
+                <img class="about__image-Redux" src="img/React Redux.svg" alt="">
+                <p class="about__name">React Redux</p>
+              </div>
+
+              <div class="about__block">
+                <img class="about__image-JS" src="img/Node JS.svg" alt="">
+                <p class="about__name">Node JS</p>
+              </div>
+
+              <div class="about__block">
+                <img class="about__image-Git" src="img/Git.svg" alt="">
+                <p class="about__name">Git</p>
+              </div>
+
+              <div class="about__block">
+                <img class="about__image-Mongo" src="img/Mongo DB 1.svg" alt="">
+                <p class="about__name">Mongo DB</p>
+              </div>
+
+              <div class="about__block">
+                <img class="about__image-MySQL" src="img/MySQL.svg" alt="">
+                <p class="about__name">MySQL</p>
+              </div>
+
+            </div>
+          </div>
+
+
+          <div class="column-lg-5">
+              <img class="about__picture" src="img/picture.jpg" alt="">
+          </div>
+
+      </div>
+    </div>
+  </section>
+
+
+
+
+
+
+ <!-- Block aside -->
+  <section class="aside">
+    <div class="container">
+      <div class="row">
+
+        <div class="column-lg-6">
+          
+
+          <div class="aside__figure">
+            <div class="aside__circle"></div>
+            <div class="aside__circle"></div>
+            <div class="aside__circle"></div>
+          </div>
+
+
+          
+          <div class="aside__square">
+
+            <div class="aside__title">Кому подойдет этот курс</div>
+             
+            
+            <ul>
+
+              <div class="aside__block">
```

Comment:
incorrect nesting.
Check this https://caninclude.glitch.me/caninclude?child=div&parent=ul

Reply:
+